shared/datastore_medium_test.go

```diff
@@ -44,3 +44,68 @@ func TestLoadTestRuns(t *testing.T) {
 	assert.Equal(t, 1, len(loaded))
 	assert.Equalf(t, key.IntID(), loaded[0].ID, "ID field should be populated.")
 }
+
+func TestLoadTestRuns_Experimental_Only(t *testing.T) {
+	testRuns := TestRuns{
+		TestRun{
+			ProductAtRevision: ProductAtRevision{
+				Product: Product{
+					BrowserName:    "chrome",
+					BrowserVersion: "63.0",
+					OSName:         "linux",
+				},
+				Revision: "1234567890",
+			},
+			ResultsURL: "/static/chrome-63.0-linux-summary.json.gz",
+			CreatedAt:  time.Now(),
+		},
+		TestRun{
+			ProductAtRevision: ProductAtRevision{
+				Product: Product{
+					BrowserName:    "chrome-experimental",
+					BrowserVersion: "63.0",
+					OSName:         "linux",
+				},
+				Revision: "1234567890",
+			},
+			ResultsURL: "/static/chrome-experimental-63.0-linux-summary.json.gz",
+			CreatedAt:  time.Now(),
+		},
+		TestRun{
+			ProductAtRevision: ProductAtRevision{
+				Product: Product{
+					BrowserName:    "chrome",
+					BrowserVersion: "64.0",
+					OSName:         "linux",
+				},
+				Revision: "1234567890",
+			},
+			ResultsURL: "/static/chrome-64.0-linux-summary.json.gz",
+			CreatedAt:  time.Now(),
+			Labels:     []string{"experimental"},
+		},
+	}
+
+	i, err := aetest.NewInstance(&aetest.Options{StronglyConsistentDatastore: true})
+	assert.Nil(t, err)
+	defer i.Close()
+	r, err := i.NewRequest("GET", "/api/run?product=chrome-66.0&labels=experimental", nil)
```

Comment:
Nit: These params aren't actually used, so I'd make the URL simpler to avoid confusion for the test reader.